.github/workflows/tg-unit-tests.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
           echo "TT_METAL_HOME=$(pwd)" >> $GITHUB_ENV
       - name: Build UMD device and tests
         run: |
-          cmake -B build -G Ninja
+          cmake -B build -G Ninja -DTT_UMD_BUILD_TESTS=ON
```

Comment:
What the heck - why are we building the tests on the TG machine? We should be downloading build artifacts. Sounds like an issue to write up to do after

Reply:
Can you write up said issue? I don't know what a TG machine is yet.
Eventually, we don't want to be building UMD tests at all in tt-metal CI, because UMD Is an external third party library, and they are starting to run their own CI.